src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,23 @@
+package racingcar.view;
+
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수는 모두 대문자가 컨벤션 입니다:)
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```

Reply:
수정했습니다!